### ISSUE TITLE
alicloud_mongo_instances data has empty mongos list

### ISSUE THREAD

**Issue by chanind** [terraform-providers/terraform-provider-alicloud#612]
I have a mongo DB instance on alicloud which has 2 nodes in replica mode. When I try to load this info via `alicloud_mongo_instances` it loads data about the mongo instance but the list of nodes under `mongos` is empty. In my state file I see the following, which is incorrect:

```
"instances.0.mongos.#": "0",
```

Without being able to load any info about the replica nodes I can't construct a mongo URL for my services to use.

### Terraform Version
Terraform v0.11.11
provider.alicloud v1.26.0

### Affected Resource(s)
Please list the resources as a list, for example:
- data.alicloud_mongo_instances

### Terraform Configuration Files
```hcl
data "alicloud_mongo_instances" "example" {
  name_regex = "test"
}
```

**Comment by AJenpan** [comment]
hi, I will begin to fix it next week.

**Comment by xiaozhu36** [comment]
HI @chanind This issue has been fixed by 1.42.0. Please check it.
